src/Simulation/QSharpFoundation/Random/Intrinsic.qs

```diff
@@ -17,7 +17,7 @@ namespace Microsoft.Quantum.Random {
     /// probability.
     ///
     /// # Remarks
-    /// Fails if `max <= min`.
+    /// Fails if `max < min`.
```

Comment:
We shouldn't edit this before the C# runtime support has been added for it.

Reply:
Reverted this change.